### PR TITLE
AO3-6493 Notify wrangling supervisors via email when a wrangler changes their username

### PR DESCRIPTION
**Pull request by Bilka2**
## Issue

https://otwarchive.atlassian.net/browse/AO3-6493

## Purpose

Create a Tag Wrangling Supervisor mailer and add a mail to the Tag Wrangler Supervisors when a tag wrangler changes their username.

## Testing Instructions

See Jira.

## Credit
Bilka (he/him)

### CODE REVIEW

File: config/config.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ RETURN_ADDRESS: 'do-not-reply@example.org'
 SPAM_ALERT_ADDRESS: 'abuse-discuss@example.org'
 SPAM_THRESHOLD: 15
 ADMIN_ADDRESS: 'admin@example.org'
+TAG_WRANGLER_SUPERVISORS_ADDRESS: 'tagwranglers-personnel@transformativeworks.org'
```
Comment (Bilka2) [inline config/config.yml:37]: Should this also have `@example.org` like the other default addresses?

Comment (sarken) [reply config/config.yml:37]: Yes please! We prefer placeholder values in the config file.

File: test/mailers/previews/tag_wrangling_supervisor_mailer_preview.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class TagWranglingSupervisorMailerPreview < ApplicationMailerPreview
+  # Sent to tag wrangling supervisors when a tag wrangler changes their username
+  def wrangler_username_change_notification
+    TagWranglingSupervisorMailer.wrangler_username_change_notification("stack inserter", "bulk inserter")
```
Comment (weeklies) [inline test/mailers/previews/tag_wrangling_supervisor_mailer_preview.rb:4]: Not a big deal, but spaces aren't allowed in usernames.

File: config/locales/mailers/en.yml [hunk 3]
```diff
@@ -74,6 +74,9 @@ en:
           addressed: Hi, %{name}!
           unaddressed: Hi!
         introductory: Hello from the Archive of Our Own!
+        tag_wrangling_supervisor:
+          html: Hello %{tag_wrangler_supervisors},
```
Comment (Bilka2) [inline config/locales/mailers/en.yml:78]: Would it be okay to use the standard informal greeting instead? That would be `Hi, Tag Wrangler Supervisors!`. I didn't want to make the email different from the Jira issue, but it seems a bit awkward to have two so similar greetings.

Comment (sarken) [reply config/locales/mailers/en.yml:78]: I checked with TW and the standard one is indeed fine! They also send their thanks for working on this; they've very excited.

**Review by brianjaustin — APPROVED** [review]

**Pull request merged**